build/uuid-redirects.json

```
@@ -552,5 +552,6 @@
     "Compendium.pf2e.spell-effects.Item.yke7fAUDxUNzouQc": "Compendium.pf2e.spell-effects.Item.Spell Effect: Elemental Gift",
     "Compendium.pf2e.spell-effects.Item.yq6iu0Qxg3YEbb6s": "Compendium.pf2e.spell-effects.Item.Spell Effect: Elemental Gift",
     "Compendium.pf2e.spell-effects.Item.zdpTPf157rXl3tEJ": "Compendium.pf2e.spell-effects.Item.Spell Effect: Clawsong",
-    "Compendium.pf2e.the-slithering-bestiary.Actor.idj4GBmsLUNUbk9r": "Compendium.pf2e.pathfinder-bestiary.Actor.Alchemical Golem"
-}
+    "Compendium.pf2e.the-slithering-bestiary.Actor.idj4GBmsLUNUbk9r": "Compendium.pf2e.pathfinder-bestiary.Actor.Alchemical Golem",
+    "Compendium.pf2e.spell-effects.Item.4HJJdogS751Z04lD": "Compendium.pf2e.spell-effects.Item.Effect: Tail Lash"
+}
```

```suggestion
    "Compendium.pf2e.bestiary-effects.Item.4HJJdogS751Z04lD": "Compendium.pf2e.spell-effects.Item.Effect: Tail Lash"
}

```
I gave you the wrong path, wasn't paying attention there. This should be inserted in alphabetical order, but I can just do a sort on the file later. I'm sure it needs it anyways.